src/lamp_py/tableau/jobs/filtered_hyper.py

```diff
@@ -70,7 +62,7 @@ def update_parquet(self, _: None) -> bool:
         # this relies on the FilteredHyperJob persisting across runs - currently it is
         # constructed on library load, but if it is reconstructed on each run_hyper() invocation,
         # this will no longer hold.
-        if self.first_run:
+        if self.first_run:  # type: ignore
```

Comment:
suggestion: having to make mypy ignore this simple bool variable is very odd, and likely points to some other issue with the data structure. I'd make all attempts to remove this, if possible.